WalletWasabi/WabiSabi/Models/InputRoundSignaturePair.cs

```diff
@@ -0,0 +1,22 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.WabiSabi.Models
+{
+	public class InputRoundSignaturePair
+	{
+		public InputRoundSignaturePair(OutPoint input, byte[] roundSignature)
+		{
+			Input = input;
+			RoundSignature = roundSignature;
+		}
+
+		public OutPoint Input { get; }
+
+		public byte[] RoundSignature { get; }
```

Comment:
I don't remember why we decided to use the byte arrays instead of `ECDSASignature`. Do you?

Reply:
We're using `BitcoinWitPubKeyAddress.VerifyMessage` currently in `ChaumianCoinJoinController` and there's no  `ECDSASignature` overload.